src/datreant/core/trees.py

```diff
@@ -130,8 +130,12 @@ def touch(self):
         """Make file if it doesn't exist.
 
         """
-        self.makedirs()
-        self.path.touch()
+        if os.name == 'nt':
```

Comment:
Looking at the contents of `touch_me`, it's not obvious to me why this is necessary. It looks like it does exactly the same thing?